docs/bios-settings.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Redirecting to https://fwupd.github.io/libfwupdplugin/bios-settings.html</title>
```

Comment:
eigh? Not sure I get the *why*

Reply:
I was thinking we don't need to include libfwupdplugin in our URLs then.  We just do a front landing URL that will redirect us.  That's how we do HSI too.

Reply:
Ahh, gotcha.

Reply:
decided to drop it